### PR TITLE
B3: Migrate User-facing Subscription Mgmt Views

### PR DESCRIPTION
**Pull request by biyeun**
@dimagi/product 
@millerdev / anyone

will need QA
screen shot 2016-03-01 at 2 42 43 am
screen shot 2016-03-01 at 2 43 13 am
screen shot 2016-03-01 at 2 43 41 am
screen shot 2016-03-01 at 3 38 33 am
screen shot 2016-03-01 at 3 38 46 am
screen shot 2016-03-01 at 3 39 01 am
screen shot 2016-03-01 at 3 39 07 am
screen shot 2016-03-01 at 3 39 16 am
screen shot 2016-03-01 at 3 39 26 am
screen shot 2016-03-01 at 3 39 38 am

### CODE REVIEW

File: corehq/apps/domain/forms.py [hunk 1]
```diff
@@ -1307,11 +1313,12 @@ def __init__(self, account, domain, creating_user, current_subscription,
...
                 'confirm_legal',
             ),
-            FormActions(
-                crispy.HTML('<a href="%(url)s" style="margin-right:5px;" class="btn">%(title)s</a>' % {
-                    'url': reverse(DomainSubscriptionView.urlname, args=[self.domain]),
-                    'title': _("Cancel"),
-                }),
+            hqcrispy.FormActions(
+                hqcrispy.LinkButton(
+                     _("Cancel"),
```
Comment (dimagimon) [inline corehq/apps/domain/forms.py]: E126 continuation line over-indented for hanging indent

File: corehq/apps/domain/forms.py [hunk 2]
```diff
@@ -1763,13 +1780,19 @@ def __init__(self, domain, web_user, *args, **kwargs):
...
-                    }
+                crispy.Div(
+                    crispy.Div(
+                        crispy.HTML(_(
+                            '<p><i class="fa fa-info-circle"></i> Clicking "Update" will set '
+                            'up the subscription in CommCareHQ to one of our standard '
+                            'contracted plans.<br/> If you need to set up a non-standard plan, '
+                            'please email %(accounts_email)s.</p>') % {
+                                'accounts_email': settings.ACCOUNTS_EMAIL,
+                            }
```
Comment (dimagimon) [inline corehq/apps/domain/forms.py]: E123 closing bracket does not match indentation of opening bracket's line

**Comment by amsagoff** [comment]
:credit_card: :moneybag: :heart_eyes:

**Comment by biyeun** [comment]
on staging

File: corehq/apps/style/tests/test_compress_command.py [hunk 3]
```diff
@@ -39,6 +39,8 @@ def _assert_valid_import(self, line, filename):
             self.assertNotRegexpMatches(tag[0], line.strip(), tag[1])
         if 'src' not in line and 'href' not in line:
             return
+        if not 'static' in line:
```
Comment (dimagimon) [inline corehq/apps/style/tests/test_compress_command.py]: E713 test for membership should be 'not in'

**Comment by millerdev** [comment]
:+1:

File: corehq/apps/style/tests/test_compress_command.py [hunk 4]
```diff
@@ -37,6 +37,8 @@ def _assert_valid_import(self, line, filename):
             return
         for tag in DISALLOWED_TAGS:
             self.assertNotRegexpMatches(tag[0], line.strip(), tag[1])
+        if not 'static' in line:
```
Comment (dimagimon) [inline corehq/apps/style/tests/test_compress_command.py]: E713 test for membership should be 'not in'

**Comment by biyeun** [comment]
QA complete... cc @benrudolph

**Pull request merged**